### PR TITLE
Bots emagged by humans will no longer interface with silicons.

### PR DESCRIPTION
**Pull request by xxalpha**
Fixes https://github.com/tgstation/-tg-station/issues/3993
 - Bots emagged by humans will no longer interface with silicons (hacked bots will)
 - Replaced Emag() proc with emag_act()

### CODE REVIEW

**Comment by GunHog** [comment]
Odd. I do not know who changed it to where silicons could. The issue you reference is older than Botcall, but that was my intent to prevent them interfacing with human emagged bots, as well as ones with the wireless link turned off. (along with PDAs)

**Comment by AnturK** [comment]
It wasn't changed. Silicons could open the window but not use any functions. This makes them unable to get any response at all.

File: code/modules/mob/living/simple_animal/bot/bot.dm [hunk 1]
```diff
@@ -140,7 +140,7 @@
 /mob/living/simple_animal/bot/proc/explode()
 	qdel(src)
 
-/mob/living/simple_animal/bot/proc/Emag(mob/user) //Master Emag proc. Ensure this is called in your bot before setting unique functions.
+mob/living/simple_animal/bot/emag_act(mob/user)
```
Comment (tkdrg) [inline code/modules/mob/living/simple_animal/bot/bot.dm]: looks like you accidentally deleted the leading / here

Comment (xxalpha) [inline code/modules/mob/living/simple_animal/bot/bot.dm]: relative pathing compiling this shit

**Pull request merged**